docs/Community/governance/README.md

```diff
@@ -19,16 +19,23 @@ Members
 
 | Name                 | Company     | Email                         | GitHub Alias   |
 |----------------------|-------------|-------------------------------|----------------|
-| Aeva van der Veen    | Microsoft   | aeva@dont.io                  | AevaOnline     |
+| Aeva Black           | Microsoft   | aeva@dont.io                  | AevaOnline     |
+| Anand Krishnamoorthi | Microsoft   | anakrish@microsoft.com        | anakrish       |
+| Dave Thaler          | Microsoft   | dthaler@microsoft.com         | dthaler        |
+| Radhika Jandhyala    | Microsoft   | radhikaj@microsoft.com        | radhikaj       |
+| Simon Leet           | Microsoft   | simon.leet@microsoft.com      | CodeMonkeyLeet |
+
+
+Emeritus Members
+----------------
+
+| Name                 | Company     | Email                         | GitHub Alias   |
```

Comment:
Can we also add Jordan here?

Reply:
We can also do it in a different PR

Reply:
My intent was only to move those members who replied on the CGC list.

I have no concerns about adding Jordan, but for tracking purposes, I'd suggest a separate commit / PR.